data/top-pypi-packages-365-days.json

```diff
@@ -0,0 +1 @@
count":1999960,"project":"astral"},{"download_count":1996773,"project":"python-box"},{"download_count":1992382,"project":"python-openstackclient"},{"download_count":1987939,"project":"toposort"},{"download_count":1984547,"project":"httptools"},{"download_count":1980989,"project":"asynctest"},{"download_count":1978811,"project":"pycalverter"},{"download_count":1975990,"project":"django-mptt"},{"download_count":1974600,"project":"nameparser"},{"download_count":1974472,"project":"geomet"},{"download_count":1974084,"project":"rtree"},{"download_count":1970886,"project":"gax-google-logging-v2"},{"download_count":1967604,"project":"openapi-spec-validator"},{"download_count":1966141,"project":"simpleeval"},{"download_count":1965371,"project":"gax-google-pubsub-v1"},{"download_count":1964155,"project":"pympler"},{"download_count":1957946,"project":"pint"},{"download_count":1954321,"project":"django-celery-results"},{"download_count":1950586,"project":"oauth2"},{"download_count":1947313,"project":"collections-extended"},{"download_count":1943588,"project":"dparse"},{"download_count":1937747,"project":"azure-mgmt-botservice"},{"download_count":1935888,"project":"facebook-business"},{"download_count":1932910,"project":"django-localflavor"},{"download_count":1931470,"project":"slackweb"},{"download_count":1919103,"project":"azure-eventhub"},{"download_count":1918652,"project":"django-braces"},{"download_count":1917375,"project":"fake-useragent"},{"download_count":1916732,"project":"python-engineio"},{"download_count":1904465,"project":"django-countries"},{"download_count":1901273,"project":"ptvsd"},{"download_count":1899393,"project":"orderedmultidict"},{"download_count":1897121,"project":"jwcrypto"},{"download_count":1895022,"project":"azure-mgmt-security"},{"download_count":1893082,"project":"awacs"},{"download_count":1889385,"project":"azure-functions-devops-build"},{"download_count":1884376,"project":"locket"},{"download_count":1882404,"project":"ctutlz"},{"download_count":1875062,"project":"snapshottest"},{"download_count":1874184,"project":"pdfkit"},{"download_count":1870591,"project":"scapy"},{"download_count":1869037,"project":"opencensus-context"},{"download_count":1862753,"project":"flask-mail"},{"download_count":1860985,"project":"intervaltree"},{"download_count":1856012,"project":"azure-mgmt-sqlvirtualmachine"},{"download_count":1853788,"project":"azure-mgmt-kusto"},{"download_count":1853245,"project":"luigi"},{"download_count":1852083,"project":"pylru"},{"download_count":1848356,"project":"sklearn-pandas"},{"download_count":1846838,"project":"pydantic"},{"download_count":1845633,"project":"email-validator"},{"download_count":1844376,"project":"pyquery"},{"download_count":1841139,"project":"django-oauth-toolkit"},{"download_count":1839835,"project":"memory-profiler"},{"download_count":1839825,"project":"jupyterlab-server"},{"download_count":1835726,"project":"sqlalchemy-migrate"},{"download_count":1832053,"project":"retry-decorator"},{"download_c
```

Comment:
Why minify this? Minified JSON is good if it's part of a frequently used service. But here I'd rather be able to see what's in the list.

Reply:
I thought that it would be better to not waste memory on this. But then again, it's only around 120KBs more.